logistics_project/apps/malawi/warehouse/report_views/stock_status.py

```diff
@@ -1,9 +1,14 @@
 from __future__ import unicode_literals
 from builtins import range
 import json
+import folium
```

Comment:
I think this library needs to be added to `requirements.in` / `requirements.txt`